tso-server/src/main/java/org/apache/omid/tso/RetryProcessorImpl.java

```diff
@@ -69,7 +72,8 @@
     private final Meter noCTFoundMeter;
 
     @Inject
-    RetryProcessorImpl(MetricsRegistry metrics,
+    RetryProcessorImpl(TSOServerConfig config,
+                       MetricsRegistry metrics,
```

Comment:
Same comment as above. Inject strategy instead of tso config:

```java
    @Inject
    RetryProcessorImpl(@Named("RetryStrategy") WaitStrategy strategy,
                       MetricsRegistry metrics,
                       CommitTable commitTable,
                       ReplyProcessor replyProc,
                       Panicker panicker,
                       ObjectPool<Batch> batchPool)
            throws InterruptedException, ExecutionException, IOException {

        // ------------------------------------------------------------------------------------------------------------
        // Disruptor initialization
        // ------------------------------------------------------------------------------------------------------------

        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("retry-%d").build();
        this.disruptorExec = Executors.newSingleThreadExecutor(threadFactory);

        this.disruptor = new Disruptor<>(EVENT_FACTORY, 1 << 12, disruptorExec, SINGLE, strategy);
```